### DISCUSSION TITLE
DAI 2023-2024 - Practical work 3 - Jonas Troeltsch - Sarah Jallon - Jeremiah Steiner

### DISCUSSION THREAD

**Discussion by BetterBlood** [heig-vd-dai-course/heig-vd-dai-course#163] [Show and tell]
Tower defense, multicast enemies attacker (different types), unicast clients that can either defend or heal the tower (server).

**Comment by sarsurgithub** [comment]
last commit 6d69485cccd5974a946b05f11727baf5d9ad5a6b

**Reply by sarsurgithub** [reply]
repo: https://github.com/BetterBlood/DAI-PW3
